.README/rules/tag-lines.md

```diff
@@ -21,9 +21,13 @@ Use with "always" to indicate the number of lines to require be present.
 Use with "always" to indicate the normal lines to be added after tags should
 not be added after the final tag.
 
-##### `dropEndLines` (defaults to `false`)
+##### `startLines` (defaults to `undefined`)
```

Comment:
Does `startLines: 1` together with `newline-after-description` in total add two empty lines after a description block with this?

Reply:
Good question, lol. That rule should probably also be dropped with this PR. :-)

Setting `startLines` to 0 is like that rule's `"never"` option and setting it to 1 is like the rule's `"always"` option, so no need for the rule anymore.

Reply:
Although we could move the behavior to `newline-after-description` to keep more backward compatibility, we would then need to refer people using `sort-tags` to two additional rules if they wanted initial tag and final tag whitespace control (`tag-lines` and `newline-after-description`). So I think it would be best to remove the `newline-after-description` rule since we're making breaking changes here anyways.

Reply:
The dropping of `newline-after-description` is now a part of the PR.